docs/setup/settings.asciidoc

```diff
@@ -592,6 +592,9 @@ The `server.xsrf.whitelist` setting requires the following format:
 [cols="2*<"]
 |===
 
+| `status.xsrf.disableProtection:`
+ | Setting this to `true` will completely disable XSRF protection in Kibana. This is not recommended. *Default: `false`*
```

Comment:
What do you think about spelling it out for folks who might not be familiar, or who are googling for the term?


```suggestion
 | Setting this to `true` will completely disable Cross-site request forgery protection in Kibana. This is not recommended. *Default: `false`*
```

Reply:
Good idea.